acra/configuring-maintaining/metrics/_index.md

```diff
@@ -1,4 +1,75 @@
+
+
+## Metrics configuration
+
+### Command line flags
+
+* `--incoming_connection_prometheus_metrics_string=<url>`
+
+  URL (tcp://host:port) which will be used to expose Prometheus metrics (`<URL>/metrics` address to pull metrics).
+
+
+### Exposed metrics
+
+Acra can expose standard set of metric [`types`](https://prometheus.io/docs/concepts/metric_types/) for AcraServer, AcraConnector, and AcraTranslator. 
+Additional metrics registration info could be found in [`prometheus.go`](https://github.com/cossacklabs/acra-Q12021/blob/master/cmd/acra-server/common/prometheus.go) files located in `cmd` folder for each corresponded Acra component.
+
+
+
+|                            Metric                                   |              Description                   |                                     Type                                   | 
+| ------------------------------------------------------------------- | ------------------------------------------ | -------------------------------------------------------------------------- | 
+|  `acra{server|translator|connector}_connections_total`              | Number of connections to database          | [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)     | 
+|  `acra{server|translator|connector}_connections_processing_seconds` | Time of connection processing              | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acra{server|translator}_request_processing_seconds`               | Time of request processing                 | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acraserver_response_processing_seconds`                           | Time of response processing                | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acra_api_encryptions_total`                                       | Number of encryptions data to AcraStruct   | [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)     |
+|  `acra{server|translator|connector}_version_major`                  | Major number of version                    | [`Gauge`](https://prometheus.io/docs/concepts/metric_types/#gauge)         |
+|  `acra{server|translator|connector}_version_minor`                  | Minor number of version                    | [`Gauge`](https://prometheus.io/docs/concepts/metric_types/#gauge)         |
+|  `acra{server|translator|connector}_version_patch`                  | Patch number of version                    | [`Gauge`](https://prometheus.io/docs/concepts/metric_types/#gauge)         |
+|  `acra{server|translator|connector}_build_info`                     | Build number                               | [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)     |
+
+
+
+### Metrics example
```

Comment:
can we show separate examples of metrics that exports each of services? example for AcraServer, for AcraTranslator, for AcraConnector? they are different and it will be helpful for users to see them all here and it allow to compare our list with their